WordPress/src/main/java/org/wordpress/android/models/ReaderPost.java

```diff
@@ -273,7 +273,7 @@ private static void assignTagsFromJson(ReaderPost post, JSONObject jsonTags) {
             int postCount = jsonThisTag.optInt("post_count");
             if (postCount > popularCount) {
                 nextMostPopularTag = mostPopularTag;
-                mostPopularTag = JSONUtils.getStringDecoded(jsonThisTag, "name");
+                mostPopularTag = JSONUtils.getStringDecoded(jsonThisTag, "display_name");
```

Comment:
Wondering if this should be also be `slug`.  It looks like in places like `getTagForDisplay` the string passed will be the slug for the current tag.